hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java

```diff
@@ -3029,24 +3044,39 @@ public boolean addAcl(OzoneObj obj, OzoneAcl acl) throws IOException {
    */
   @Override
   public boolean removeAcl(OzoneObj obj, OzoneAcl acl) throws IOException {
-    if(isAclEnabled) {
-      checkAcls(obj.getResourceType(), obj.getStoreType(), ACLType.WRITE_ACL,
-          obj.getVolumeName(), obj.getBucketName(), obj.getKeyName());
-    }
-    // TODO: Audit ACL operation.
-    switch (obj.getResourceType()) {
-    case VOLUME:
-      return volumeManager.removeAcl(obj, acl);
-    case BUCKET:
-      return bucketManager.removeAcl(obj, acl);
-    case KEY:
-      return keyManager.removeAcl(obj, acl);
-    case PREFIX:
-      return prefixManager.removeAcl(obj, acl);
+    Map<String, String> auditMap = obj.toAuditMap();
+    auditMap.put(OzoneConsts.ACLS, acl.toString());
```

Comment:
We can move this audit log to the common method.
As this is common code for all ACL requests.

`auditAcl(OzoneObj ozoneObj, OzoneAcl ozoneAcl, OMAction omAction, Exception exception)`

Reply:
@bharatviswa504 I actually spotted a typo because of your comment.
removeAcl & addAcl will use `auditMap.put(OzoneConsts.ACL, acl.toString())`
setAcl will use `auditMap.put(OzoneConsts.ACLS, acls.toString())`
getAcl will not have such a line.
The difference between first 2 is that setAcl can have more than one Acl in question hence using 'ACLS' to indicate multiple.
Let me know if you think we still need to move this line to common method and avoid such detailed classification.

Reply:
Discussed offline, how we can make this audit for acls in to a common method, which can help HA code path.

And also can you file a Jira to make code changes to address this in HA code path also.